docs/concepts/resource_flavor.md

```diff
@@ -0,0 +1,87 @@
+# ResourceFlavor object
+
+Resources in a cluster are typically not homogeneous. Resources could differ in:
+- pricing and availability (ex: spot vs on-demand VMs)
+- architecture (ex: x86 vs ARM CPUs)
+- brands and models (ex: Radeon 7000 vs Nvidia A100 vs T4 GPUs)
+
+A ResourceFlavor is an object that represents these resource variations and
+allows you to associate them with node labels and taints.
+
+**Note**: If your cluster is homogeneous, you can use an [empty ResourceFlavor](#empty-resourceflavor)
+instead of adding labels to custom ResourceFlavors.
+
+A sample ResourceFlavor looks like the following:
+
+```yaml
+apiVersion: kueue.x-k8s.io/v1alpha2
+kind: ResourceFlavor
+metadata:
+  name: spot
+nodeSelector:
+  instance-type: spot
+taints:
+- effect: NoSchedule
+  key: spot
+  value: "true"
+```
+
+You can use the `.metadata.name` to reference a ResourceFlavor from a
+ClusterQueue in the `.spec.resources[*].flavors[*].name` field.
+
+## ResourceFlavor labels
+
+**Requires Kubernetes 1.23 or newer**
+
+To associate a ResourceFlavor with a subset of nodes of your cluster, you can
+configure the `.nodeSelector` field with matching node labels that uniquely identify
+the nodes. If you are using [cluster autoscaler](https://github.com/kubernetes/autoscaler/tree/master/cluster-autoscaler)
+(or equivalent controllers), make sure it is configured to add those labels when
```

Comment:
```suggestion
(or equivalent controllers), make sure the controller is configured to add those labels when
```

I'm just assuming "it" refers to "the controller". I highly recommend you change "it" for the subject name to avoid confusion.